src/root/rootReducer.ts

```diff
@@ -28,15 +27,14 @@ import speciesSelector from 'src/content/app/species-selector/state/speciesSelec
 import entityViewer from 'src/content/app/entity-viewer/state/entityViewerReducer';
 import speciesPage from 'src/content/app/species/state/index';
 
-const createRootReducer = (history: any) =>
+const createRootReducer = () =>
   combineReducers({
     browser,
     customDownload,
     communication,
     inAppSearch,
     genome,
     global,
-    router: connectRouter(history),
```

Comment:
I am wondering how this will affect #663 as it depends on `goBack` and `goForward` in `connected-react-router`.

Reply:
Okay found the answer to this and also now recall we discussed about this before Christmas last year: https://github.com/remix-run/react-router/blob/5730e28a7472e3a1b967d95c5c4cc643c570d5c7/docs/upgrading/v5.md